### PR TITLE
Add --type flag to `mc admin console`

### PR DESCRIPTION
**Pull request by poornas**
to allow filtering error logs by type

### CODE REVIEW

File: cmd/admin-console.go [hunk 1]
```diff
@@ -37,6 +37,11 @@ var adminConsoleFlags = []cli.Flag{
 		Usage: "show last n log entries",
 		Value: 10,
 	},
+	cli.StringFlag{
+		Name:  "type, t",
+		Usage: "list error logs by type.Valid options are '[minio,application,all]'",
```
Comment (krisis) [inline cmd/admin-console.go]:
```suggestion
		Usage: "list error logs by type. Valid options are '[minio, application, all]'",
```

File: cmd/admin-console.go [hunk 2]
```diff
@@ -37,6 +37,11 @@ var adminConsoleFlags = []cli.Flag{
 		Usage: "show last n log entries",
 		Value: 10,
 	},
+	cli.StringFlag{
+		Name:  "type, t",
+		Usage: "list error logs by type.Valid options are '[minio, application, all]'",
```
Comment (krisis) [inline cmd/admin-console.go]:
```suggestion
		Usage: "list error logs by type. Valid options are '[minio, application, all]'",
```

**Comment by kannappanr** [comment]
@poornas PTAL CI failure

**Comment by poornas** [comment]
> @poornas PTAL CI failure

that's because of the go.mod replace directive. the minio/minio PR needs to be merged first.

**Comment by kannappanr** [comment]
@poornas Can you please resolve the conflicts

**Pull request merged**